public/locales/es/common.json

```diff
@@ -0,0 +1,272 @@
+{
+  "actions": {
+    "ok": "OK"
+  },
+  "balances": {
+    "apy": "APY",
+    "apyRange": "Rango de APY",
+    "balance": "Saldo",
+    "depositValue": "Cantidad depositada",
+    "earned": "Ganancias",
+    "marketCap": "Capitalización de mercado",
+    "max": "Max",
+    "pending": "Pendiente",
+    "picklePerBlock": "{{ amount }} PICKLEs son creados en cada bloque.",
+    "picklePerSecond": "{{ amount }} PICKLEs son creados cada segundo.",
+    "poolBalance": "Cantidad acumulada",
+    "poolSize": "Valor acumulado",
+    "poolSizeTooltip": "Total ETH/PICKLE pool value on Uniswap.",
+    "ratio": "Ratio",
+    "staked": "Invertido",
+    "totalApy": "APY total",
+    "totalSupply": "Cantidad total",
+    "totalValueLocked": "Valor total depositado",
+    "tvl": "TVL",
+    "valueStaked": "Cantidad invertida",
+    "walletBalance": "Saldo en la cartera",
+    "yourApy": "Tu APY",
+    "yourBalance": "Tu saldo"
+  },
+  "connection": {
+    "blockNumber": "Este es el número de bloque actual. Esta página se actualiza con cada nuevo bloque.",
+    "changeNetwork": "Cambiar de red",
+    "coinbase": "Coinbase",
+    "connect": "Connect Wallet",
```

Comment:
```suggestion
    "connect": "Conecta tu Cartera",
```

Reply:
Thanks! I believed it was another wallet name